crates/torii/grpc/src/server/mod.rs

```diff
@@ -398,7 +400,7 @@ impl DojoWorld {
             .collect::<Vec<_>>()
             .join(" OR ");
 
-        if table == EVENT_MESSAGES_HISTORICAL_TABLE {
+        if table == EVENT_MESSAGES_HISTORICAL_TABLE || table == ENTITIES_HISTORICAL_TABLE {
```

Comment:
Something like this yields an empty `where` clause:
```rust
let response = world_client
    .retrieve_entities(Query {
        clause: None,
        limit: 10,
        offset: 0,
        dont_include_hashed_keys: false,
        order_by: vec![],
        entity_models: vec![],
        entity_updated_after: 0,
    }, !historical)
    .await?;
```
Which results in an invalid query in the `if` 
```sql
SELECT COUNT(*) FROM entities_historical
JOIN entity_model ON entities_historical.id = entity_model.entity_id
WHERE 
GROUP BY entities_historical.event_id
```

Reply:
Fixed